CONTRIBUTING.md

```diff
@@ -1,32 +1,135 @@
-The Janitor sits atop a number of other projects, and those are where
-most of the interesting things happen. You may want to check out one of them.
+[The Janitor](https://github.com/jelmer/janitor) sits atop a number of other
+projects, and those are where most of the interesting things happen.
+You may want to check out one of them.
 They're probably also easier to setup and run, unlike the Janitor.
 
-Environment
-===========
+## Environments
 
-Mostly you can use pip to install Python-based dependencies. In addition to
-those, you'll also want to install various other bits of software.
+It is recommended to use Debian Testing as the base OS/chroot.
+
+### Development Environment
+
+You'll want to install various bits of software.
 On a Debian-based OS, run:
 
 ```console
 $ sudo apt install \
     cargo \
+    gcc \
     git \
-    libapt-pkg-dev \
+    libpython3-dev \
+    libssl-dev \
+    pkg-config \
     protobuf-compiler \
-    python3-pip \
-    python3-venv \
-    rustc \
-    swig
+    python3-gpg
 ```
 
-For example, to create a development environment:
+<!--
+In order (Package, command, error):
+  - git                  $ git clone https://github.com/jelmer/janitor.git     # -bash: git: command not found
```

Comment:
What is this comment for?

Reply:
Was a note of understanding/remembering why each of those packages are 'core' and what happens if you don't have them.

Reply:
This makes the README harder to read for those not reading the rendered version. Can you drop this?

Reply:
Done